RaveBusinessLogic/VBET.xml

```diff
@@ -1,145 +1,50 @@
-                <Node label="Plausible Valley Bottom Extent" xpath="Geopackage/Layers/Vector[@id='VBET_50']" type="polygon" id="vbet50" symbology="vbet50" transparency="40" />
-              </Children>
-            </Node>
-
-          </Children>
-          </Node>
-        
-        </Children>
-      </Node>
-      <Node label="Inputs">
-        <Children collapsed="true">
-          <Node label="Hydrography">
-            <Children collapsed="true">
-              <Node xpathlabel="Name" xpath="Intermediates/Geopackage/Layers/Vector[@id='VBET_NETWORK']" type="line" symbology="vbet_network" />
-              <Node label="VBET Network with Bankfull" xpath="Intermediates/Geopackage/Layers/Vector[@id='VBET_NETWORK']" type="line" id="network" symbology="bankfull_flowlines" />
-              <Node label="Full NHD HR+ Drainage Network" xpath="Inputs/Geopackage/Layers/Vector[@id='FLOWLINES']" type="line" symbology="flow_lines" />
-              <Node label="Perennial Drainage Network" xpath="Inputs/Geopackage/Layers/Vector[@id='FLOWLINES']" type="line" symbology="nhdperrenial" />
-              <Node label="Intermittent Drainage Network" xpath="Inputs/Geopackage/Layers/Vector[@id='FLOWLINES']" type="line" symbology="nhdintermittent" />
-              <Node label="Ephemeral Drainage Network" xpath="Inputs/Geopackage/Layers/Vector[@id='FLOWLINES']" type="line" symbology="nhdephemeral" />
-              <Node label="Network - Upstream Drainage Network" xpath="Inputs/Geopackage/Layers/Vector[@id='FLOWLINES']" type="polygon" symbology="upstreamdrainagearea" />
-              <Node label="River Polygons (NHD Area)" xpath="Inputs/Geopackage/Layers/Vector[@id='FLOW_AREA']" type="polygon" symbology="nhdarea" />
-              <Node xpathlabel="Name" xpath="Inputs/Geopackage/Layers/Vector[@id='CATCHMENTS']" type="polygon" symbology="" />
-              <Node xpathlabel="Name" xpath="Intermediates/Raster[@id='CHANNEL_BUFFER_RASTER']" type="raster" symbology="" />
-              <Node xpathlabel="Name" xpath="Intermediates/Raster[@id='FLOW_AREA_RASTER']" type="raster" symbology="" />
-            </Children>
-          </Node>
-          <Node label="Topography">
-            <Children collapsed='true'>
-              <Node label="Slope Analysis" xpath="Inputs/Raster[@id='SLOPE_RASTER']" type="raster" symbology="slope" transparency="40" />
-              <Node label="Height Above Nearest Drainage" xpath="Intermediates/Raster[@id='HAND_RASTER']" type="raster" symbology="hand" transparency="40" />
-              <Node label="Digital Elevation Model" xpath="Inputs/Raster[@id='DEM']" type="raster" symbology="dem" transparency="40" id="dem" />
-              <Node xpathlabel="Name" xpath="Inputs/Raster[@id='HILLSHADE']" type="raster" symbology="" id="hillshade" />
+              <Node label="Estimated Active Channel" xpath="Geopackage/Layers/Vector[@id='VBET_100']" type="polygon" symbology="vbet100outline" id="vbet100outline"/>

```

Comment:
Get rid of this one (we won't use it as we have the channel area polygon already).